datadog/api/api_client.py

```diff
@@ -0,0 +1,213 @@
+# stdlib
+import logging
+import time
+
+# datadog
+from datadog.api import _api_version, _max_timeouts, _backoff_period
+from datadog.api.exceptions import ClientError, ApiError, HttpBackoff, \
+    HttpTimeout, ApiNotInitialized
+from datadog.api.http_client import RequestClient
+from datadog.util.compat import json, is_p3k
+
+
+log = logging.getLogger('dd.datadogpy')
+
+
+class APIClient(object):
+    """
+    """
+    # HTTP transport parameters
+    _backoff_period = _backoff_period
+    _max_timeouts = _max_timeouts
+    _backoff_timestamp = None
+    _timeout_counter = 0
+    _api_version = _api_version
+
+    def __init__(self):
```

Comment:
This seems to be the only non-static method: you could remove it, set `_http_client` as a class variable, and `_get_http_client` the first time `submit` or another method is called. (or in the definition, or during the import, I'm not sure what's the best method)